### PR TITLE
CNF 1483 RN: Describing the net queues feature in the 4.8 release notes

### PR DESCRIPTION
**Pull request by kquinn1204**
Release notes for 4.8 describing net queues feature CNF 1483
Preview Link: https://deploy-preview-31016--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes.html#ocp-4-8-scale

### CODE REVIEW

**Comment by kquinn1204** [comment]
Published RN https://deploy-preview-31016--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes.html#ocp-4-8-scale

**Comment by kquinn1204** [comment]
@MarSik @yanirq I have drafted this small paragraph https://deploy-preview-31016--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes.html#ocp-4-8-scale for the release note. Let me know if you are happy with it, keeping it simple as key config, etc in the linked doc.

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -227,6 +227,14 @@ For more information, see xref:../scalability_and_performance/what-huge-pages-do
 [id="ocp-4-8-scale"]
 === Scale
 
+[id="ocp-4-8-reducing-nic-queues-using-the-performance-addon-operator"]
+==== Reducing NIC using the Performance Addon Operator
+
+The Performance Addon Operator allows you to adjust the Network Interface Card (NIC) queue count for each network device by configuring the performance profile. Device network queues allow packets to be distributed among different physical queues and each queue gets a separate thread for packet processing.
```
Comment (jeana-redhat) [inline release_notes/ocp-4-8-release-notes.adoc:233]:
```suggestion
The Performance Addon Operator allows you to adjust the Network Interface Card (NIC) queue count for each network device by configuring the performance profile. Device network queues allow packets to be distributed among different physical queues, and each queue gets a separate thread for packet processing.
```

File: release_notes/ocp-4-8-release-notes.adoc [hunk 2]
```diff
@@ -227,6 +227,14 @@ For more information, see xref:../scalability_and_performance/what-huge-pages-do
...
+
+For Data Plane Development Kit (DPDK) based workloads it is important to reduce the NIC queues to just the number of reserved or housekeeping CPUs to ensure the desired low latency is achieved.
```
Comment (jeana-redhat) [inline release_notes/ocp-4-8-release-notes.adoc:235]:
```suggestion
For Data Plane Development Kit (DPDK) based workloads, it is important to reduce the NIC queues to only the number of reserved or housekeeping CPUs to ensure the desired low latency is achieved.
```

**Review by jeana-redhat — COMMENTED** [review]
Good addition :)

**Pull request merged**